libmamba/ext/solv-cpp/src/pool.cpp

```diff
@@ -93,10 +93,17 @@ namespace solv
 
     namespace
     {
+// This function is only used in `assert()` expressions
+// That's why it might get reported as unused in Release builds
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
I explained above why this happens.
It's completely ok to have a function which is only used in `assert` statements.
So, I silence this warning/error